luma/core/interface/serial.py

```diff
@@ -270,15 +270,19 @@ class spi(bitbang):
     """
     def __init__(self, spi=None, gpio=None, port=0, device=0,
                  bus_speed_hz=8000000, cs_high=False, transfer_size=4096,
-                 gpio_DC=24, gpio_RST=25):
+                 gpio_CS=None, gpio_DC=24, gpio_RST=25):
```

Comment:
why not add this as last argument? i'd prefer that

Reply:
I was just going with the order in the bitbang class, but either way is fine for me, changed :)